openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -15708,6 +15710,47 @@ OS:Controller:MechanicalVentilation,
        \type object-list
        \object-list DesignSpecificationZoneAirDistributionNames
 
+OS:ZoneControl:ContaminantController,
+       \min-fields 8
+  A1, \field Handle
+       \type handle
+       \required-field
+  A2 , \field Name
+       \required-field
+  A3 , \field Controlled Zone Name
```

Comment:
Well reversing the relationship may be worse. In that case you could have multiple zones referencing the same controller.